src/k8s-engine/hyperkit.ts

```diff
@@ -455,6 +455,10 @@ export default class HyperkitBackend extends events.EventEmitter implements K8s.
         this.emit('service-changed', services);
       });
       this.activeVersion = desiredVersion;
+      // Trigger kuberlr to ensure there's a compatible version of kubectl in place
+      await childProcess.spawnFile(resources.executable('kubectl'), ['config', 'current-context'],
```

Comment:
This works fine; I wonder if `cluster-info` would be more useful for debugging though (since it's going to the log anyway).

Reply:
Made the change -- that info is potentially more useful.